server/methods/accounts/accounts.app-test.js

```diff
@@ -22,7 +23,8 @@ before(function () {
 
 describe("Account Meteor method ", function () {
   let shopId;
-  const fakeUser = Factory.create("account");
```

Comment:
this spec had been using User and Account interchangeably. Further, it was creating new accounts here and there, which wind up using the same User (i.e., the userId of all of them was the same). I am fairly certain this resulted in further false-positives